addon/services/raven.js

```diff
@@ -144,7 +142,7 @@ export default Service.extend({
     if (this.get('isRavenUsable')) {
       Raven.captureMessage(...arguments);
     } else {
-      throw new Error(message);
+      Ember.Logger.debug(message);
```

Comment:
afaik `Ember.Logger` is somewhat deprecated and it is recommended to use `console.log` (and the related methods) directly